packages/react-core/src/next/components/Dropdown/DropdownGroup.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+import { MenuGroupProps, MenuGroup } from '../../../components/Menu';
+
+export interface DropdownGroupProps extends Omit<MenuGroupProps, 'ref'> {
+  /** Anything which can be rendered in a dropdown group. */
+  children: React.ReactNode;
+  /** Classes applied to root element of dropdown list */
+  className?: string;
+  /** Label of the dropdown group */
+  label?: string;
+  /** Group label heading level.*/
+  labelHeadingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
It's something I'd have to maybe look into a bit more, but I'm unsure about heading elements within a dynamically rendered component like the dropdown. Mainly because the heading would be unnoticed unless the dropdown is opened, and at that point a user may have already scanned the page via headings (assuming something like VoiceOver's rotor menu, at least).

Reply:
I can remove it.  It is a prop on the menu that the dropdown extends.  we do not have that prop on the old dropdown.

Reply:
Ah gotcha. In that case I'd be fine with the prop remaining/being added back in, especially since without it (and without anything being passed into that internal MenuGroup inside this Dropdown next component) it'll trickle down to the internal MenuGroup using an `h1` by default it looks like.

(thank you for being willing to remove it only to have to possibly add it back in! 😭 )

Reply:
@thatblindgeye h1 is what we have it set to in our current implementation of the dropdown.  So I think that is fine.  If there is a use case to change the heading level we can always add the prop back later.